indico/modules/events/contributions/templates/management/_contribution_list.html

```diff
@@ -184,7 +184,7 @@
                             <td class="i-table actions-column">
                                 <div class="group right entry-action-buttons vertical-aligner">
                                     <a href="#" class="icon-edit i-button-icon js-dialog-action"
-                                       data-href="{{ url_for('.manage_update_contrib', contrib) }}"
+                                       data-href="{{ url_for('.manage_update_contrib', contrib, flash = True) }}"
```

Comment:
`flash=true` (kwargs have no space around the `=` and jinja uses/prefers lowercase true/false/none, even though uppercase also works)